concordia/views.py

```diff
@@ -276,12 +277,33 @@ class AccountProfileView(LoginRequiredMixin, FormView):
 
     def get_context_data(self, *args, **kwargs):
         ctx = super().get_context_data(*args, **kwargs)
-        ctx["transcriptions"] = (
-            Transcription.objects.filter(user=self.request.user)
-            .select_related("asset__item__project__campaign")
-            .order_by("asset__pk", "-pk")
-            .distinct("asset")
+
+        transcriptions = Transcription.objects.filter(user=self.request.user)
+        items = Item.objects.filter(pk__in=transcriptions.values("asset__item__pk"))
+        items = items.select_related("project", "project__campaign")
+        items = items.order_by("project__campaign__title", "project__title", "item_id")
```

Comment:
I think we're missing another order by field here, for `transcription.created_on desc`